packages/steveo/src/consumers/kafka.ts

```diff
@@ -106,10 +106,12 @@ class KafkaRunner
         }
 
         this.logger.debug('Start subscribe', c.topic, message);
-
-        // @ts-ignore
-        const context = parsed.value.context ?? null;
-        await task.subscribe(parsed, context);
+        const { _meta = {}, ...data } = parsed.value;
+        const context = {
+          ...(parsed.value.context ?? {}),
+          ..._meta,
+        };
```

Comment:
I merged `parsed.value.context` but I am not sure it is really needed.
Any suggestions @sugendran @tjcoyoca @johndagostino?

Reply:
@lmagalhaes I think this would change if we rebase main, we have changes around kafka's consumer implementation here: https://github.com/ordermentum/steveo/pull/787/files

Reply:
I don't think you need to do that give the changes in the producer. `parsed.value` should just be the _meta and the actual message payload